hack/pre-run-e2e.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+source "${REPO_ROOT}"/hack/util.sh
+
+# variable define
+KUBECONFIG_PATH=${KUBECONFIG_PATH:-"${HOME}/.kube"}
+MAIN_KUBECONFIG=${MAIN_KUBECONFIG:-"${KUBECONFIG_PATH}/karmada.config"}
+HOST_CLUSTER_NAME=${HOST_CLUSTER_NAME:-"karmada-host"}
+KARMADA_APISERVER_CLUSTER_NAME=${KARMADA_APISERVER_CLUSTER_NAME:-"karmada-apiserver"}
+MEMBER_CLUSTER_KUBECONFIG=${MEMBER_CLUSTER_KUBECONFIG:-"${KUBECONFIG_PATH}/members.config"}
+MEMBER_CLUSTER_1_NAME=${MEMBER_CLUSTER_1_NAME:-"member1"}
+MEMBER_CLUSTER_2_NAME=${MEMBER_CLUSTER_2_NAME:-"member2"}
+PULL_MODE_CLUSTER_NAME=${PULL_MODE_CLUSTER_NAME:-"member3"}
+
+export VERSION="latest"
+export REGISTRY="swr.ap-southeast-1.myhuaweicloud.com/karmada"
+
+CERT_DIR=${CERT_DIR:-"${HOME}/.karmada"}
+ROOT_CA_FILE=${CERT_DIR}/server-ca.crt
+
+# load interpreter webhook example image
+kind load docker-image "${REGISTRY}/karmada-interpreter-webhook-example:${VERSION}" --name="${HOST_CLUSTER_NAME}"
```

Comment:
Where and how is the image built?  Is it built from source?

Reply:
Because it run in CI, so the image has build in front.

Do we need to put it in this place? I think it may be not suitable.

Reply:
Get it, from here:
https://github.com/karmada-io/karmada/blob/6f4b65a99f2617f467de47509ee8d6a44b11c3be/hack/local-up-karmada.sh#L109

Reply:
> Do we need to put it in this place? I think it may be not suitable.

No, I didn't mean that, just a question.